tests/optim/test_optim.py

```diff
@@ -100,16 +101,16 @@ def guide():
         for _ in range(num_steps):
             svi.step()
         if epoch == 1:
-            loc = pyro.param('loc')
-            scale = pyro.param('scale')
+            loc = pyro.param('loc').unconstrained()
+            scale = pyro.param('scale').unconstrained()
             opt = scheduler.optim_objs[loc].optimizer
             assert opt.state_dict()['param_groups'][0]['lr'] == 0.02
             assert opt.state_dict()['param_groups'][0]['initial_lr'] == 0.01
             opt = scheduler.optim_objs[scale].optimizer
             assert opt.state_dict()['param_groups'][0]['lr'] == 0.02
             assert opt.state_dict()['param_groups'][0]['initial_lr'] == 0.01
             assert abs(pyro.param('loc').item()) > 1e-5
-            assert abs(pyro.param('scale').item()) - 0.5 > 1e-5
+            assert abs(pyro.param('scale').item() - 0.5) > 1e-5
```

Comment:
@jpchen - I was getting scale has invalid values validation error so added a positive constraint.

Reply:
i noticed that error but never got around to fixing it. thanks for fixing it 😄